assignments/arrays.js

```diff
@@ -63,34 +63,62 @@ let inventory = [{"id":1,"car_make":"Lincoln","car_model":"Navigator","car_year"
 
 // ==== Challenge 1 ====
 // The dealer can't recall the information for a car with an id of 33 on his lot. Help the dealer find out which car has an id of 33 by logging the car's year, make, and model in the console log provided to you below:
-console.log(`Car 33 is a *car year goes here* *car make goes here* *car model goes here*` );
-
-
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === 33) {
+        console.log(inventory[32]);
+        // console.log(inventory[i].car_make);
+        // console.log(inventory[i].car_model);
+        // console.log(inventory[i].car_year);
+    }
+}
 
 // ==== Challenge 2 ====
 // The dealer needs the information on the last car in their inventory.  What is the make and model of the last car in the inventory?  Log the make and model into the console.
-let lastCar = 0;
-console.log();
+for (let i = 0; i < inventory.length; i++) {
+    if (inventory[i].id === inventory[inventory.length-1].id) {
+        console.log(inventory[i].car_make);
+        console.log(inventory[i].car_model);
```

Comment:
Consider using more descriptive console.logs to recognize them when they are outputted to the console.
Like this: `console.log('make: ', inventory[i].car_make);`
You can also chain elements in a console.log function following the first element by a coma:
`console.log('first', 'second', 'third', etc....);`